examples/custom-tests/README.md

```diff
@@ -0,0 +1,25 @@
+### Example: Running custom tests on built images
+
+This example shows how to run
+[custom tests]
+on newly built images in your skaffold dev loop. Tests are associated with single
```

Comment:
Please explain more how test dependencies is different than source dependencies.
1) when test file changes, no build will happen but tests would get re-run